substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2532,6 +2533,16 @@ mod runtime {
 	pub type Revive = pallet_revive::Pallet<Runtime>;
 }
 
+impl TryFrom<RuntimeCall> for pallet_revive::Call<Runtime> {
+	type Error = ();
+
+	fn try_from(value: RuntimeCall) -> Result<Self, Self::Error> {
+		match value {
+			RuntimeCall::Revive(call) => Ok(call),
+			_ => Err(()),
+		}
+	}
+}
```

Comment:
Where is this used?

Reply:
here https://github.com/paritytech/polkadot-sdk/blob/pg/revive-rpc-fixes-v2/substrate/frame/revive/src/evm/runtime.rs?plain=1#L135

Reply:
Got it thanks

Reply:
yea we could update the proc macro to generate it for us